packages/xo-web/src/common/usage/index.js

```diff
@@ -2,30 +2,39 @@ import _ from 'intl'
 import classNames from 'classnames'
 import PropTypes from 'prop-types'
 import React, { cloneElement } from 'react'
-import { compact, sum } from 'lodash'
+import { compact, sumBy } from 'lodash'
 
 import Tooltip from '../tooltip'
 
-const Usage = ({ total, children }) => {
+const Usage = ({ total, children, link }) => {
   const limit = total / 400
-  const othersValues = compact(
+  const othersProps = compact(
     React.Children.map(children, child => {
       const { value } = child.props
-      return value < limit && value
+      return value < limit && child.props
     })
   )
-  const othersTotal = sum(othersValues)
-  const nOthers = othersValues.length
+  const othersTotal = sumBy(othersProps, 'value')
+  const nOthers = othersProps.length
   return (
     <span className='usage'>
       {nOthers > 1 ? (
         <span>
           {React.Children.map(
             children,
             (child, index) =>
-              child.props.value > limit && cloneElement(child, { total })
+              child.props.value > limit &&
+              cloneElement(child, {
+                href: typeof link === 'function' ? link(child.props.id) : link,
+                total,
+              })
```

Comment:
You also need to handle the case where `nOthers >= 1`. (can't put my comment there so I'm putting it here)